skbio/metadata/tests/test_mixin.py

```diff
@@ -76,5 +79,40 @@ def __deepcopy__(self, memo):
         self._positional_metadata_constructor_ = ExamplePositionalMetadataMixin
 
 
+class TestIntervalMetadataMixin(unittest.TestCase, ReallyEqualMixin,
+                                IntervalMetadataMixinTests):
+    def setUp(self):
+        super()._set_up()
```

Comment:
I view it the opposite way: note that `IntervalMetadataMixinTests` is not inherited from `TestCase`, so  naming it `setUp` may cause more confusion. More importantly, call `super().setUp` will call `TestCase.setUp()` rather than `IntervalMetadataMixinTests.setUp` if I name `_set_up` to `setUp`.